js_machine_front/src/scenes/news/news.tsx

```diff
@@ -1,18 +1,48 @@
-import React, { memo } from 'react';
-import Background from './images/news.jpg';
-
-const sectionStyle = {
-  height: '100vh',
-  backgroundImage: `url(${Background})`,
-  backgroundRepeat: 'no-repeat',
-  backgroundSize: 'cover',
-  backgroundPosition: 'center',
-};
-
-export const News: React.FC = memo(() => {
-  return (
-    <div style={ sectionStyle }>
-      <div className="body"/>
-    </div>
-  );
-});
+import React, { PureComponent } from 'react';
+import { DigestView } from 'components/DigestView/DigestView';
+
+const sectionStyle = {
+  backgroundColor: 'grey',
+  paddingTop: 100,
+};
+
+interface State {
+  openDigest: boolean;
+}
+
+export class News extends PureComponent<{}, State> {
+
+  constructor(props: {}) {
+    super(props);
+
+    this.state = {
+      openDigest: false,
+    };
+  }
+
+  openDigest = () => {
+    this.setState({ openDigest: true });
+  }
+
+  closeDigest = () => {
+    this.setState({ openDigest: false });
+  }
+
+  exitKeyPressHandler = (e: KeyboardEvent) => {
+    if (e.keyCode === 27) {
+      this.closeDigest();
+    }
+  }
+
+  render(): JSX.Element {
+    return (
+      <div style={sectionStyle}>
+        <h1>News</h1>
+        <button onClick={this.openDigest}>Open digest</button>
+        <DigestView isOpen={this.state.openDigest}
```

Comment:
please create a new scene for digest view!!!

Reply:
this is not the same as news, you need to create a separate route

Reply:
okey